docs/howto/sizing_recommendations.md

```diff
@@ -12,6 +12,12 @@ title: Sizing recommendations
     databases) and then memory consumption is dependent on how "wide"
     are the metrics gathered.
 
+!!! Notice
+    In high-load environments, some metrics may return thousands of rows per monitoring target at each fetch. 
+    So, sharing the collection interval for these metrics between all sources can cause a huge memory load.
```

Comment:
Good one! I would add about LIMIT clause for such metrics. I mean there is no sense in having 1K recommendations anyway. One always start with first 10